daq/ipaddr_test.py

```diff
@@ -60,6 +62,15 @@ def _dhcp_port_toggle_test(self):
         self.host.connect_port(True)
         self._ip_callback = self._next_test
 
+    def _dhcp_lease_wait_test(self):
+        if not self.host.connect_port(False):
+            self.log('disconnect port not enabled')
```

Comment:
what happens in this case.... is the error silently ignored?  Does it show up as skip? fail? pass? fatal error?

Reply:
@grafnu If this test is not successful in ipaddr.py, no wait time occurs or a port toggle resulting in no DHCP request. Within the DHCP test code, this will return a fail on faux interface 2 (as expected anyway) and also a fail on faux interface 1. If USI were to fail I do not believe there is a way for this to be detected in the DHCP test code due to the separation between docker tests and USI - so an ideal skip case would not be possible.

Reply:
My question is what happens after logging 'disconnect port is not enabled' occurs -- does it just log the error message then nothing else? What's the observable result of this condition?  The log file should explain why something happened, but it should be obvious by other behaviors that there was a problem in the first place.